### ISSUE TITLE
Use SVG icons without the viewBox

### ISSUE THREAD

**Issue by dandv** [shshaw/SVGvsFont#3]
From [a comment](https://css-tricks.com/icon-fonts-vs-svg/#comment-1580671) in the original blog post:

> In your third comment where you say: “Each icon has double the amount of markup as SVG (`<svg viewBox=”0 0 34 32″ class=”fa fa-glass”><use xlink:href=”#fa-glass”></use></svg>`) versus Icon Font (`<i class=”fa fa-glass”></i>`)”, the icon fonts can easily be applied via the :before and/or :after pseudo elements, thus, reducing even more the markup needed to implement icon fonts.

**Comment by dandv** [comment]
While the above is true, SVG icons can be used without the viewBox part if the `viewBox` attribute is included in the [`symbol`](https://css-tricks.com/svg-symbol-good-choice-icons/) definition. We use the approach below [in production](https://idorecall.com):

``` html
<head>
<svg id="svg-source" style="display: none;" version="1.1" xmlns="http://www.w3.org/2000/svg">
  <defs>
   <symbol id="fa-facebook" viewBox="0 0 1792 1792">
      <title>Facebook icon from Font Awesome</title>
      <svg><path d="M1579 128q35 0 60 25t25 60v1366q0 35-25 60t-60 25h-391v-595h199l30-232h-229v-148q0-56 23.5-84t91.5-28l122-1v-207q-63-9-178-9-136 0-217.5 80t-81.5 226v171h-200v232h200v595h-735q-35 0-60-25t-25-60v-1366q0-35 25-60t60-25h1366z"></path></svg>
    </symbol>
</defs></svg></head>

<body>
<a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A//iDoRecall.com/?sref">
  <svg><use xlink:href="#fa-facebook"></use></svg>
</a>
</body>
```

**Comment by shshaw** [comment]
You're correct. I've been using `viewBox`-less `<svg>`s with a `viewBox` on the referenced symbol for a while now. The only issue is you must include proper sizing for the containing `<svg>` element, or the browser default size (at least in Chrome) of 300x150 will be applied, often leaving space around the icon or sizing it strangely.

A few options are:
- Adding `viewBox` to container SVG
- Adding `width` and `height` to the container SVG
- Sizing container SVG with CSS

[Here's a demo](http://codepen.io/shshaw/pen/zGQzYK)
